fpsim/locations/ethiopia.py

```diff
@@ -384,6 +425,19 @@ def lactational_amenorrhea():
     return lactational_amenorrhea
 
 
+def lactational_amenorrhea_region():
+    '''
+    Returns an array of the percent of breastfeeding women by month postpartum 0-11 months who meet criteria for LAM, , stratified by region
+    Uses 2016 Ethiopia DHS individual recode (v025) for region and V024 for urban to produce subnational estimates
+    '''
+    lactational_amenorrhea_region = pd.read_csv(thisdir / 'ethiopia' / 'subnational' / 'lam_region.csv')
+    lactational_amenorrhea_region['region'] = lactational_amenorrhea_region[:, 0] # Return region names
```

Comment:
In order to access the columns of data in the lactational_amenorrhea dataframe we've loaded above, you actually need to use the following syntax: lactational_amenorrhea_region.iloc[:,0]. I would define a dictionary above this data retrieval like so: lam_dict = {} and then use the syntax I mentioned to correct the data extraction, and finally return the lam_dict (distinct name from the dataframe you initially read in)

Reply:
This parameter now includes a dictionary that correctly facilitates data retrieval.